cfme/tests/configure/test_logs.py

```diff
@@ -105,23 +105,13 @@ def test_provider_log_updated(appliance, provider, log_exists):
     assert log_before != log_after, "Log hashes are the same"
 
 
-@pytest.mark.meta(blockers=[BZ(1633656,
```

Comment:
This bug is still valid

Reply:
Thanks for double checking me on these, when I was making the change I was creating BZ objects and checking their blocks state, a few without 5.10 tagged clones slipped through.